romfs/source/fighter/common/param/common.prcxml

```diff
@@ -50,7 +50,7 @@
   <float hash="hit_stop_delay_flick_mul">5</float>
   <int hash="hit_stop_delay_flick_frame">2</int>
   <int hash="hit_stop_delay_flick_max_count">50</int>
-  <float hash="hit_stop_delay_auto_mul">2.5</float>
+  <float hash="hit_stop_delay_auto_mul">1.7</float>
```

Comment:
Was ASDI distance meant to be 1.7 here? If not bump this down to 1.5 like in the PR description, otherwise you should update the PR description and LGTM

Reply:
Resolved over discord (https://discord.com/channels/659964948365049887/950979364634382376/1024842038992175224), LGTM